contracts/0.4.24/Lido.sol

```diff
@@ -455,29 +444,32 @@ contract Lido is StETH, AragonApp {
         // EL values
         uint256 _withdrawalVaultBalance,
         // decision
-        uint256 _withdrawalsReserveAmount,
+        uint256 _newBufferedEtherReserveAmount,
```

Comment:
Maybe `_newBufferedEtherDepositReserve`?
or even  `_newBufferedEtherNonDepositReserve`
or `_newBufferNonDepositReserve`, `_newNonDepositBufferReserve`
cause it's reserve not for deposits, but on contrary for NOT deposits

In general, like when there is no need to go through usage and see for what "reserve" is, if it has unique usage.

Reply:
Not sure. Generally, I don't like too long names. Sometimes it's better to follow and check the overall logic and docs instead of trying to figure out the exact meaning if it's not clear from the start.